docs/nftfair/introduction.md

```diff
@@ -0,0 +1,59 @@
+# Introduction
+
+Automata NFTFair is A NFT launch platform designed around principles of fairness and efficacy for a community-led experience is how we would describe NFTFair. Metadata exploits pre-sniping are all predatory practices that make it unfair for the average user to get their hands on a new collection, much less obtain rare or lucrative items.
+
+We want to make every NFT journey begins with fairness — clearly a non-negotiable attribute. More than just kicking off a positive feedback loop between trust and adoption in the metaverse, it’s imperative for any NFT launch that seeks to do right by its community.
+
+# About the fairdrop?
+
+## Network:
+
+You can launch a fairdrop in most of the EVM-compatible networks. Such as ETH, BSC, Polygon, Moonriver, except for the layer 2 network. We don't support the layer 2 network for now. Due to the expensive transaction gas fee in the ETH network, we recommend you launch your fairdrop in BSC.
```

Comment:
why layer2 is not supported?

Reply:
I think it is better to tell users/projects that we support EVM-compatible networks, and list the already supported networks

Reply:
The issue with Layer 2 is due to the complexity of gas cost calculation, which significantly impacts Conveyor.

The current gas price calculation on the contract would not work for rollup chains, i.e. Optimism and Arbitrum.

Reply:
List of supported network (In accordance to XATA):
- BSC
- Polygon
- Moonriver

Reply:
This platform is compatible with most EVM networks. We currently support the hosting of Fairdrop events on the following networks:
BSC
Polygon
Moonriver
Ethereum

We are working hard to expand the list of supported networks and if your intended launch network is not included in our supported list, please send us an email at <email>.